packages/docprovider/src/tokens.ts

```diff
@@ -13,27 +13,9 @@ export const IDocumentProviderFactory = new Token<IDocumentProviderFactory>(
  */
 export interface IDocumentProvider {
   /**
-   * Resolves to true if the initial content has been initialized on the server. false otherwise.
+   * Returns a Promise that resolves when renaming is ackownledged.
```

Comment:
Not sure what it means for renaming to be acknowledged. Can we describe it a bit more?